src/main/java/oof/Storage.java

```diff
@@ -49,6 +50,39 @@
         }
     }
 
+    /**
+     * Reads the threshold previously saved to hard disk.
+     *
+     * @return threshold    Integer of threshold.
+     * @throws OofException if file does not exist.
+     */
+    public int readThreshold() throws OofException {
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(PATH_THRESHOLD));
+            String line;
+            line = reader.readLine();
+            int threshold;
+            threshold = Integer.parseInt(line);
+            return threshold;
+        } catch (IOException e) {
+            throw new OofException("File Unavailable!");
```

Comment:
Unnecessary to throw an exception here, just create a default config file

Reply:
Issue resolved.